### PR TITLE
Upgrade org.dbunit:dbunit Trunk 5536

### PR DESCRIPTION
**Pull request by georgejunior**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5536

## Checklist: I completed these to help reviewers :)

- [x ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by ODORA0** [comment]
Have you looked at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

File: pom.xml [hunk 1]
```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.6.3</version>
```
Comment (ODORA0) [inline pom.xml]: Does the ticket also require this upgrade? You pushed to the same base branch. You can follow this https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

**Comment by georgejunior** [comment]
> You should squash your commits. you could look at this https://wiki.openmrs.org/display/docs/Git-Squashing+commits?

but i have only one commit

**Comment by ODORA0** [comment]
Have you looked at the comment after that one?

File: pom.xml (hunk 1 above)
Comment (georgejunior) [reply pom.xml]: > Does the ticket also require this upgrade? You pushed to the same base branch. You can follow this https://wiki.openmrs.org/display/docs/Using+Gits

but it's like this page you are linking me to doesn't exist

Comment (ODORA0) [reply pom.xml]: It's this https://wiki.openmrs.org/display/docs/Using+Git that's a wrong link

**Comment by ODORA0** [comment]
Think it looks good now
cc @mozzy11

File: pom.xml [hunk 2]
```diff
@@ -500,7 +500,7 @@
 			<dependency>
 				<groupId>org.dbunit</groupId>
 				<artifactId>dbunit</artifactId>
-				<version>2.5.4</version>
+				<version>2.5.4 </version>
```
Comment (samuelmale) [inline pom.xml]: @georgejunior is this an upgrade or just whitespace?

Comment (georgejunior) [reply pom.xml]: @samuelmale i first did the required changes (upgrade) then as i was making some changes, i just added the whitespace

Comment (samuelmale) [reply pom.xml]: Where is the upgrade, you mean it was already merged into core? If not, I don't any work done here @georgejunior .

Comment (ODORA0) [reply pom.xml]: So are you going to make the required update? You can look at this https://mvnrepository.com/artifact/org.dbunit/dbunit to see the latest update

Comment (HerbertYiga) [inline pom.xml]: @georgejunior  kindly look at odora's comment!!

**Comment by mozzy11** [comment]
@georgejunior  it looks fine .
cc @dkayiwa

**Comment by dkayiwa** [comment]
The commit message should include the ticket id: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by georgejunior** [comment]
> The commit message should include the ticket id: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

okay let me add it

**Comment by georgejunior** [comment]
> The commit message should include the ticket id: https://wiki.openmrs.org/display/docs/Pull+Request+Tips



> The commit message should include the ticket id: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

@dkayiwa  i have added the ticket id

**Pull request merged**